learning_assistant/views.py

```diff
@@ -95,9 +98,17 @@ def post(self, request, course_run_id):
 
         course_id = get_course_id(course_run_id)
 
-        prompt_template = render_prompt_template(request, request.user.id, course_run_id, unit_id, course_id)
+        if response_variation == ResponseVariations.GPT4_UPDATED_PROMPT:
+            gpt_model = GptModels.GPT_4o
+            template_string = getattr(settings, 'LEARNING_ASSISTANT_EXPERIMENTAL_PROMPT_TEMPLATE', '')
+        else:
+            gpt_model = GptModels.GPT_3_5_TURBO
```

Comment:
TODO: Make sure this is _actually_ the default before merging

Reply:
updated based on the default here: https://github.com/2uinc/ai-chatbot-backend/blob/7fbc9498018ad3228a17ba4e8567ebc4320baf7a/xpert/utils/openai.py#L47C54-L47C72